apps/vpnmanager/src/components/Statistics/Statistics.tsx

```diff
@@ -0,0 +1,314 @@
+import React, { useEffect, useRef, useState } from "react";
+import {
+  Box,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Paper,
+  TextField,
+  Grid,
+  Button,
+  TablePagination,
+} from "@mui/material";
+import { Section } from "@commons-ui/core";
+import { useRouter } from "next/router";
+
+import formatDateTime, { formatDate } from "@/vpnmanager/utils/formatDate";
+import { fetchJson, formatBytes } from "@/vpnmanager/utils";
+import { Link } from "@commons-ui/next";
+
+export interface Data {
+  ID: number;
+  userId: string;
+  usage: number;
+  date: string;
+  cumulativeData: number;
+  email: string;
+  createdAt: string;
+}
+
+interface Props {
+  data: Data[];
+}
+
+const Statistics: React.FC<Props> = ({ data: result }) => {
+  const router = useRouter();
+  const yesterday = new Date();
+  yesterday.setDate(yesterday.getDate() - 1);
```

Comment:
Since we already have `date-fns` [installed](https://github.com/CodeForAfrica/ui/blob/235b9a61d33bd2fca5c26af5bc2c68d78ae3da60/pnpm-workspace.yaml#L89), could we maybe use [`startOfYesterday`](https://date-fns.org/v4.1.0/docs/startOfYesterday)